packages/expo-dev-menu/package.json

```diff
@@ -54,7 +54,7 @@
     "babel-plugin-module-resolver": "^4.1.0",
     "expo-module-scripts": "^2.0.0",
     "fuse.js": "^6.4.6",
-    "graphql": "^14.2.1",
+    "graphql": "^15.3.0",
```

Comment:
Is this breaking change safe? Cf. https://github.com/graphql/graphql-js/releases/tag/v15.0.0

Reply:
@ide I have updated earlier GraphQL in Home app, and there was a bit of migration/refactor needed, but from my checks it looks like GraphQL in `dev-menu` is used very sparse, maybe there are several lines and few method/imports used (all from `graphql-tag` package). So it looks like the GraphQL is just a required dependency which is not used directly in the code.

Reply:
Being sparsely used doesn't mean it's safe to upgrade. I looked into graphql-tag's commit history to see when support for graphql@15 was confirmed/added. It turns out that happened [here](https://github.com/apollographql/graphql-tag/commit/7b66f0501ff05f1ea6a01385f846f09df60b8ab4), which was included in graphql-tag@2.12.0. `yarn why` says we use graphql-tag@2.12.15 everywhere so we have the updated version and bumping the graphql version is safe, but we need to understand why dependency upgrades are safe before doing them.

Reply:
@ide Yup, I agree. Thank you for the check and dependencies relation analysis!